src/newusers.c

```diff
@@ -1203,6 +1203,13 @@ int main (int argc, char **argv)
 		lines     = reallocf (lines,     sizeof (lines[0])     * nusers);
 		usernames = reallocf (usernames, sizeof (usernames[0]) * nusers);
 		passwords = reallocf (passwords, sizeof (passwords[0]) * nusers);
+		if (lines == NULL || usernames == NULL || passwords == NULL) {
+			fprintf (stderr,
+			         _("%s: line %d: %s\n"),
+			         Prog, line, strerror(errno));
+			errors++;
```

Comment:
Any reason not to just fail_exit here()?

Reply:
Just to be consistent with the rest of the code in the same function.

Reply:
If you think it's simpler to ffail_exit(), we might want to do it everywhere in that file.  That would probably simplify a lot of code.  And we can do that in many other places.  For now, I have a bunch of patches in my local repo that will imnprove use of allocation functions, making it safer and more consistent.

Reply:
Since it's going to exit at the bottom of that fn anyway, I do think it's better to exit immediately.  There isn't any partially written state to clean up, right?

Reply:
> Since it's going to exit at the bottom of that fn anyway, I do think it's better to exit immediately.

Agree.  If one line fails, I think it's safer to exit immediately.  Otherwise, some error might carry on, and result in some unexpected behavior.

> There isn't any partially written state to clean up, right?

It seems not.  `continue;` goes directly to the next input line, so the current line seems completely resolved.  And right after the loop we exit.  So, it seems a behavior improvement to break early and break hard.  I'll open a separate PR for that.